freqtrade/plugins/pairlist/ShuffleFilter.py

```diff
@@ -46,6 +53,7 @@ def filter_pairlist(self, pairlist: List[str], tickers: Dict) -> List[str]:
         :return: new whitelist
         """
         # Shuffle is done inplace
-        self._random.shuffle(pairlist)
+        random.seed(self._seed)
+        random.shuffle(pairlist)
```

Comment:
something in these 2 lines (and the removal of the 2 lines that it replaces above) bothers me, and i "feel" like it might be introducing a bug / unwanted / unexpected behavior in certain cases - but i can't pin it down what makes me feel this way.

One thing is for sure - running `random.seed(None)` (in live mode) whenever this is called is for sure unnecessary - which i think was the reason for having this as "random object" in the first place.

I'm currently not sure about the impact to backtesting (when using multiple shuffle-filters in sequence, for example) - but i guess since it's called only once per filter it shouldn't create a problem there.